.github/workflows/start-vulnerability-updater.yaml

```diff
@@ -56,3 +56,11 @@ jobs:
           go run cmd/updater/update.go -outputDir=${{ env.ROX_PRODUCT_VERSION }}
           gsutil cp -r "${{ env.ROX_PRODUCT_VERSION }}" "gs://scanner-v4-test/"
         
+  send-notification:
```

Comment:
@tommartensen how about this?

Reply:
IIUC this will only fire if the `output-versions` step fails because of the `needs` parameter.
If you want to be notified about any failure, remove the `needs: output-versions`. 
Otherwise, LGTM